website/docs/faq.md

```diff
@@ -245,8 +245,34 @@ Because the data structures supporting all three of these need to match very car
 
 ## Security
 
-<a class="anchor" id="image-id-security"></a>
+<a class="anchor" id="dev-mode"></a>
+
+<details closed>
+<summary>
+Q:
+What is developer mode and how can I use it safely?
+</summary>
+A: We support a development-only mode for standalone risc0 projects in which code is executed but not proven. This adds efficiency to the development cycle during development stages where proving correct execution is not yet critical.
+
+A risc0 project in dev mode supports ([fake](https://docs.rs/risc0-zkvm/0.18.0/risc0_zkvm/enum.InnerReceipt.html#variant.Fake)) receipt creation and pass-through (fake) 'verification' workflows, so that dev mode may be switched on and off without impacting project workflows. In particular, receipts generated in dev mode still include public outputs written to the [journal](https://dev.risczero.com/terminology#journal).
+
+However, because the proving process is bypassed, receipts generated in dev mode will fail a standard receipt verification check. Only when the verifier is also built in dev mode will it perform pass-through 'verification' of the fake receipt.
```

Comment:
This flag is an environment variable checked at runtime rather than at build-time